common/src/main/scala/com/gu/media/model/PlutoIntegrationData.scala

```diff
@@ -16,7 +16,9 @@ case class AtomAssignedProjectMessage(
   `type`: String,
   atomId: String,
   commissionId: String,
-  projectId: String
+  projectId: String,
+  title: String,
+  user: String
```

Comment:
Can we make this `Option[String]` rather than sending `unknown_user`? As I understand, `unknown_user` is the default for Pluto at ingest point; sending it here feels like we're mixing domains.

Reply:
@akash1810 sure, done that now